traffic-advice.bs

```diff
@@ -0,0 +1,265 @@
+<pre class="metadata">
+Title: Traffic Advice
+Shortname: traffic-advice
+Status: DREAM
+Repository: buettner/private-prefetch-proxy
+Editor: Jeremy Roman, Google https://www.google.com/, jbroman@chromium.org
+Abstract: A proposal to allow site owners to advice prefetch proxies and other agents to disallow traffic.
+Markup Shorthands: css no, markdown yes
+Assume Explicit For: yes
+Complain About: accidental-2119 yes, missing-example-ids yes
+Indent: 2
+Boilerplate: omit conformance
+</pre>
```

Comment:
Add `Default Biblio Status: current` to avoid the 2016-vintage <cite>Secure Contexts</cite> reference

Reply:
Done.